src/app.ts

```diff
@@ -1,6 +1,8 @@
 import { autoinject } from "aurelia-framework";
 import { RouterConfiguration, Router } from "aurelia-router";
+import {PLATFORM} from 'aurelia-pal';
 import { I18N } from 'aurelia-i18n';
+import 'app.scss';
```

Comment:
@eikaramba any idea if `<require from="app.scss">` would work?

I will try it myself later.